src/main/webapp/app/shared/layouts/navbar/navbar.component.ts

```diff
@@ -15,6 +15,10 @@ import { ParticipationWebsocketService } from 'app/overview/participation-websoc
 import { AccountService } from 'app/core/auth/account.service';
 import { ProfileService } from 'app/shared/layouts/profiles/profile.service';
 import { LoginService } from 'app/core/login/login.service';
+import { Router, NavigationEnd, ActivatedRoute } from '@angular/router';
```

Comment:
Activated Route is defined bot never used

Reply:
good catch, thanks 👍